### PR TITLE
Timeline 22.3.1

### PR DESCRIPTION
**Pull request by ghost**
1) Updated Timeline. Added one new model(common, but without migration. It is not necessary, a model will get prop-s from ready model (in a different project). 
2) Add controller in basic Timeline Controller. 
3) Some features (some lines of jquery code for shoe real time). TagHelper.

### CODE REVIEW

File: UtopiaCity/Controllers/Timeline/TimelineBasicController.cs [hunk 1]
```diff
@@ -123,5 +132,21 @@ public async Task<IActionResult> Details(string id)
...
 
+        //VIEW DATA 
+        public ActionResult ViewModelResult()
+        {
+            CollectionDataModel dataViewModel = new CollectionDataModel();
+            
+            List<Flight> flights = _flightService.GetFlightList();
+            
+            List<Event> events = _lifeService.GetAll();
+
```
Comment (Ibraga96) [inline UtopiaCity/Controllers/Timeline/TimelineBasicController.cs:143]: var?

**Review by AnatoliyDubovtsev — APPROVED** [review]

File: UtopiaCity/Controllers/Timeline/TimelineBasicController.cs (hunk 1 above)
Comment (ghost) [reply UtopiaCity/Controllers/Timeline/TimelineBasicController.cs:143]: I did not think about it. Why not)

Comment (ghost) [reply UtopiaCity/Controllers/Timeline/TimelineBasicController.cs:143]: However, I leave a longer value List<Flights> flights. Refactoring will be later

Comment (Ibraga96) [reply UtopiaCity/Controllers/Timeline/TimelineBasicController.cs:143]: ok

File: UtopiaCity/Controllers/Timeline/TimelineBasicController.cs [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 
 namespace UtopiaCity.Controllers.Timeline
 {
     public class TimelineBasicController : Controller
     {
         private readonly TimelineService _timelineService;
-
-        public TimelineBasicController(TimelineService timelineService)
+        private readonly FlightService _flightService;
+        private readonly LifeService _lifeService;
```
Comment (crocomoth) [inline UtopiaCity/Controllers/Timeline/TimelineBasicController.cs:18]: empty line?

**Review by crocomoth — APPROVED** [review]

**Pull request merged**